src/test/pybind/test_cephfs.py

```diff
@@ -52,6 +52,13 @@ def test_statfs():
     stat = cephfs.statfs(b'/')
     assert(len(stat) == 11)
 
+@with_setup(setup_test)
+def test_statx():
+    stat = cephfs.statx(b'/',0x1, 0)
```

Comment:
These should use the constants you've defined instead of numeric values for the mask. It might not hurt to create a symlink and verify that AT_SYMLINK_NOFOLLOW works, but that's not a show stopper if you don't want to do it.

Reply:
Keep the flag of 0 because it is a common tag,
And added the AT_SYMLINK_NOFOLLOW flag to determine, and the mask uses the defined macro